src/openloco/companymgr.cpp

```diff
@@ -103,4 +111,78 @@ namespace openloco::companymgr
             }
         }
     }
+
+    // 0x004383ED
+    void updateOwnerStatus()
+    {
+        if (openloco::is_title_mode() || openloco::is_editor_mode())
+        {
+            return;
+        }
+
+        auto company = companymgr::get(_updating_company_id);
+        if (company == nullptr)
+        {
+            return;
+        }
+
+        company->var_12 += 1;
+        if ((company->var_12 & 0x7F) != 0)
```

Comment:
Is this effectively keeping track of a number of ticks to postpone updates? If so, is it reset to zero in the game action? Perhaps we should just name it some such, for now.

Reply:
From what I found, it doesn't seem to be reset. If the lower bytes are all zero, the action gets executed, otherwise it exits. `if ((company->var_12 % 128) != 0) would have the same effect, I think.